### ISSUE TITLE
TypeError: Object #<CSON> has no method 'parseCSON'

### ISSUE THREAD

**Issue by guaka** [docpad/docpad#921]
Resolved #917. Now the next hoop.

In an empty subdirectory:
```
$ node --version
v0.10.29
$ docpad run
info: Welcome to DocPad v6.70.1 (global installation: /usr/local/lib/node_modules/docpad)
notice: If everyone who saw this message donated $1/week, maintaining DocPad would become sustainable: http://docpad.org/donate
info: Contribute: http://docpad.org/docs/contribute
info: Plugins:
info: Environment: development
info: Updating the exchange...
This can take a moment...
error: Something went wrong with the action
error: An error occured:
TypeError: Object #<CSON> has no method 'parseCSON'
  at /usr/local/lib/node_modules/docpad/out/lib/docpad.js:1571:23
  at Request.callback (/usr/local/lib/node_modules/docpad/node_modules/superagent/lib/node/index.js:746:30)
  at Request.<anonymous> (/usr/local/lib/node_modules/docpad/node_modules/superagent/lib/node/index.js:135:10)
  at Request.EventEmitter.emit (events.js:95:17)
  at Stream.<anonymous> (/usr/local/lib/node_modules/docpad/node_modules/superagent/lib/node/index.js:938:12)
  at Stream.EventEmitter.emit (events.js:117:20)
  at Unzip.<anonymous> (/usr/local/lib/node_modules/docpad/node_modules/superagent/lib/node/utils.js:124:12)
  at Unzip.EventEmitter.emit (events.js:117:20)
  at _stream_readable.js:920:16
  at process._tickDomainCallback (node.js:459:13)
error: To report the above error, follow the guide at: http://docpad.org/bug-report
info: Shutting down... cya next time!
```

**Comment by balupton** [comment]
Will be fixed in next release.

**Comment by balupton** [comment]
Fixed in v6.71.0 February 19, 2015

**Comment by fizzvr** [comment]
:1:
